File: Python/Uri1015_DistanciaEntreDoisPontos.py
import math

linha = input().split()
linha2 = input().split()

x1 = float(linha[0])
y1 = float(linha[1])

x2 = float(linha2[0])
y2 = float(linha2[1])

result = math.sqrt(pow((x2-x1),2) + pow((y2-y1),2))

print('{:.4f}'.format(result))

File: Javascript/Uri1015_DistanciaEntreDoisPontos.js
var input = require("fs").readFileSync("stdin", "utf8");
var lines = input.split("\n");

var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var item1 = lines.shift().split(" ");
var item2 = lines.shift().split(" ");

var x1 = item1.shift();
var y1 = item1.shift();

var x2 = item2.shift();
var y2 = item2.shift();

var result = Math.sqrt(Math.pow((x2 - x1), 2) + Math.pow((y2 - y1), 2));

console.log(result.toFixed(4));